jsons/Policies.json

```diff
@@ -146,7 +146,7 @@
 			},
 						{
 				"name": "Discipline",
-				"uniques":["[+10]% Strength for [Land] units which have another [Military] unit in an adjacent tile","[Military] units gain [50]% more Experience from combat"],
+				"uniques":["[+10]% Strength <for [Land] units> <when adjacent to a [Meele] unit","[+50]% XP gained from combat <for [Military] units>"],
```

Comment:
Isnt it 'melee' instead of 'meele'?

Reply:
yes, fixed in https://github.com/ravignir/RekMOD/pull/43/commits/046c2aeb5a6e87abee865bab714b3680cd52f276